src/index.js

```diff
@@ -1,4 +1,3 @@
-import 'regenerator-runtime/runtime';
```

Comment:
what does the regenerator have to do with this? can you point me to some docs?

Reply:
The regenerator is only needed because the transforms of async/await code into ES5. If you don't support IE11, which ES5 browser do you support exactly? Because everything else supports async/await natively

Reply:
sounds good